build_tools/cmake/ctest_all.sh

```diff
@@ -97,6 +99,9 @@ fi
 if (( IREE_AMD_RDNA3_TESTS_DISABLE == 1 )); then
   label_exclude_args+=("^requires-gpu-rdna3$")
 fi
+if (( IREE_AMD_RDNA4_TESTS_DISABLE == 1 )); then
+  label_exclude_args+=("^requires-gpu-rdna3$")
```

Comment:
Could we this? I think we can have a flag that enabled wmma tests in general and then rely on the gfxip version specified in cmake configuration. If this is hard, we should at least add a TODO for now.

Reply:
These seem to be for controlling CI

We could get rid of the flag all together